File: RightpointLabs.Pourcast.Domain/Models/Style.cs
namespace RightpointLabs.Pourcast.Domain.Models
{
    public class Style : Entity
    {
        public Style(string id, string name)
            : base(id)
        {
            Name = name;
        }

        public string Name { get; set; }
        public string Color { get; set; }
        public string Glass { get; set; }
    }
}

File: RightpointLabs.Pourcast.Web/Scripts/app/model/style.js
define('app/model/style', ['ko'], function (ko) {
    function Style(styleJSON) {
        var self = this;

        self.id = ko.observable(styleJSON.id);
        self.name = ko.observable(styleJSON.name);
        self.color = ko.observable(styleJSON.color);
        self.glass = ko.observable(styleJSON.glass);
    };

    return Style;
});